src/webgpu/util/floating_point.ts

```diff
@@ -708,6 +709,41 @@ export abstract class FPTraits {
     return new FPInterval(this.kind, n, n);
   }
 
+  /**
+   * WGSL specifies unbounded precision:
+   *   https://www.w3.org/TR/WGSL/#floating-point-accuracy
+   * In most computations doubles (number in js) are equivalent
+   * to correctly rounded intervals. However in some cases
+   * (addition/subtraction) doubles do not provide enough numeric precision.
+   *
+   * These cases are whenever a number, x (e.g. 1.0) is added (or subtracted)
+   * from a much smaller number, y (e.g. smallest positive normal), such that
```

Comment:
I would reorder this to say that y is added or subtracted from x.
Y has the small magnitude, and X has the relatively large magnitude.